pkg/heatapi/deployment.go

```diff
@@ -128,6 +128,25 @@ func Deployment(
 				Spec: corev1.PodSpec{
 					ServiceAccountName: instance.Spec.ServiceAccount,
 					Containers: []corev1.Container{
+						{
+							Name: instance.Name + "-logs",
+							Command: []string{
+								"/bin/bash",
+							},
+							Args: []string{
+								"-c",
+								"find /var/log/httpd -type f -exec tail -n+1 -F {} +",
```

Comment:
Hey @bshephar , I'd like to point out an issue we had in the past: https://issues.redhat.com//browse/OSPRH-770.
I'm not sure you want to take it in a follow up, but basically `-log sidecar` and the other containers usually ignores `SIGTERM` on delete, which means you need to wait k8s to force kill the Pod after a given timeout.
For that reason we moved to `dumb-init`, which is not a pattern but something we did in a lot of operators.
You can take a look at manila as an example of that in case you're interested [1].

[1] https://github.com/openstack-k8s-operators/manila-operator/blob/main/pkg/manilaapi/statefulset.go#L133